### DISCUSSION TITLE
CSS/Caching issue upon deployment

### DISCUSSION THREAD

**Discussion by wrobson-lllow** [payloadcms/payload#3215] [Q&A]
I recently added a small custom field component to our application, with some very simple CSS. No issues locally, everything displayed as expected. I deployed to DEV and TEST, ran a few tests in TEST and again everything displayed as expected. Next I deployed to PROD, and went to  my supervisors work station as he had to do a few things in PROD. I noticed that while the new field component was being displayed and the functionality was working as expected it was not displaying the CSS, although the CSS was working for all other previously implemented custom components. I went back and checked TEST (still working as expected) and then DEV (same issue as PROD). A forced refresh did correct the issue, but it still seems like a possible bug.
/

**Comment by denolfe** [comment]
@wrobson-lllow Can you provide more info on your environments? 

Are you deploying your front end separately to something like Vercel? Or is it being deployed _with_ Payload?

This sounds like some sort of caching issue.

**Comment by wrobson-lllow** [comment]
The front end is being deploy with Payload via AWS.

I think you are right about the caching. However it is interesting that the issue did not happen in TEST, and that I have not previously had this issue when deploying other custom components.
